scilpy/reconst/frf.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+import logging
+import numpy as np
+from dipy.core.gradients import gradient_table
+from dipy.reconst.csdeconv import auto_response
+from dipy.segment.mask import applymask
+
+from scilpy.utils.bvec_bval_tools import (is_normalized_bvecs, normalize_bvecs,
+                                          check_b0_threshold)
+
+
+def compute_ssft_frf(data, bvals, bvecs, mask=None, mask_wm=None,
+                     fa_thresh=0.7, min_fa_thresh=0.5, min_nvox=300,
+                     roi_radius=10, roi_center=None, force_b0_threshold=False):
+    """Compute a single Fiber Response Function from a DWI.
```

Comment:
Maybe specify that this is for single shell data (or, at least, under b=1500). That the reason it's `ssst` response.

Reply:
Fixed